trie/verkle.go

```diff
@@ -230,33 +228,22 @@ func nodeToDBKey(n verkle.VerkleNode) []byte {
 // Commit writes all nodes to the trie's memory database, tracking the internal
 // and external (for account tries) references.
 func (trie *VerkleTrie) Commit(_ bool) (common.Hash, *NodeSet, error) {
```

Comment:
Noticed also that in the current replay block branch we had some `onLeaf` callback here, which isn't present.
I honored this version; but just mentioning if that's relevant.

Reply:
it's only used for stuff we don't do in verkle, so that's why it's not called. But that might change with the new DB schema, we will see very soon.